sovrin_client/client/client.py

```diff
@@ -90,21 +84,8 @@ def handlePeerMessage(self, msg):
         """
         return self.peerMsgRouter.handle(msg)
 
-    def _getOrientDbStore(self):
```

Comment:
We don't use orientdb store anymore.
Why did we need it before? Should we find a (levledb) replacement?